pool/app/pool_common/entity.ml

```diff
@@ -177,6 +177,13 @@ module SortOrder = struct
   include Core
 
   let default = Ascending
+
+  let flip = function
+    | Ascending -> Descending
+    | Descending -> Ascending
+  ;;
+
+  let to_query_parts t = [ "sort_order", show t ]
```

Comment:
**fix**. where's the definition of `sort_order` so we refer to it instead of hardcoding it again here? 🤔

Reply:
fixed